torch/_refs/__init__.py

```diff
@@ -3614,6 +3614,23 @@ def repeat(a: Tensor, *repeat_shape) -> Tensor:
     return permuted_result.reshape(target_shape)
 
 
+def _nd_to_1d(a: Tensor) -> Tensor:
+    torch._check(a.numel() > 0)
+    torch._check(a.ndim > 0)
```

Comment:
There's no reason to exclude 0-d scalar tensors here.

```
>>> torch.randn([]).view(1)
tensor([-1.5574])
```

Reply:
I have removed this guard. Note that the guard is true though because of the code in ```reshape_view_helper``` before getting here.